cpp/ps0005/150daysprogramming.cpp

```diff
@@ -31,130 +16,64 @@ int main()
-
-        }
-        else {
-            cout << "(" << num1 << ") " << op << " (" << num2 << ") " << " = " << num1 / num2;
+        case '+':
+            cout << "(" << num1 << ") " << op << " (" << num2 << ") " << " = " << num1 + num2;
             break;
-        }
-
-
-    case '%':
-        if (isSt1Float || isSt2Float) {
-            cout << "cannot execute modulo operation for floating point numbers.\n";
-            break;
-        }
-        else {
-            cout << "(" << num1 << ") " << op << " (" << num2 << ") " << " = " << (int)num1 % (int)num2;
-            break;
-        }
-    default:
-        cout << "Invalid character for operator.\n";
-
-
-
-    }
-    
     
-}
-
-void days1506() {
-    string st1, st2;
-    cout << "Enter the two operands: \n";
-    cin >> st1;
-    cin >> st2;
-    cout << "\n\n Enter the operator + - * / %: \n";
-    char op;
-    cin >> op;
-    cout << endl << endl;
-    bool isSt1Float = isFloat(st1);
-    bool isSt2Float = isFloat(st2);
-
-
-
-
-
-    float num1, num2;
-    num1 = stof(st1);
-    num2 = stof(st2);
-
-
-
-    switch (op) {
-    case '+':
-        cout << "(" << num1 << ") " << op << " (" << num2 << ") " << " = " << num1 + num2;
-        break;
-
-    case '-':
-        cout << "(" << num1 << ") " << op << " (" << num2 << ") " << " = " << num1 - num2;
-        break;
-    case '*':
-        cout << "(" << num1 << ") " << op << " (" << num2 << ") " << " = " << num1 * num2;
-        break;
-    case '/':
-        if (num1 == 0 && num2 == 0) {
-            cout << "Infinite number of solutions: (0 / 0) is Undefined Quantity.\n";
+        case '-':
+            cout << "(" << num1 << ") " << op << " (" << num2 << ") " << " = " << num1 - num2;
             break;
-        }
-        else if (num2 == 0) {
-            cout << "Math Error: cannot divide by 0.";
+        
+        case '*':
+            cout << "(" << num1 << ") " << op << " (" << num2 << ") " << " = " << num1 * num2;
             break;
+        
+        case '/':
+            if (num1 == 0 && num2 == 0) {
+                cout << "Infinite number of solutions: (0 / 0) is Undefined Quantity.\n";
+                break;
+            }
+            else if (num2 == 0) {
+                cout << "Math Error: cannot divide by 0.";
+                break;
+            }
+            else {
+                cout << "(" << num1 << ") " << op << " (" << num2 << ") " << " = " << num1 / num2;
+                break;
+            } // 1/1
+        
+        case '%':
+            if (isFloat(st1) || isFloat(st2)) {
+                cout << "cannot execute modulo operation for floating point numbers.\n";
+                break;
+            }
+            else {
+                cout << "(" << num1 << ") " << op << " (" << num2 << ") " << " = " << (int)num1 % (int)num2;
+                break;
+            } // 1/1
+            
+        default:
+            cout << "Invalid character for operator.\n"; // 1/1
+    } // 1/1

```

Comment:
- [x] used `switch` and `if` structures in a right way to handle different cases
- [x] handled division operation `/`
- [x] handled modulo operation `%`
- [x] handled invalid operator case
- [x] chose the right operation for each case
- [x] printed the result of the chosen operation